source/slang/slang-ir-insts.h

```diff
@@ -430,9 +430,14 @@ IR_SIMPLE_DECORATION(NonCopyableTypeDecoration)
 IR_SIMPLE_DECORATION(HLSLMeshPayloadDecoration)
 IR_SIMPLE_DECORATION(GlobalInputDecoration)
 IR_SIMPLE_DECORATION(GlobalOutputDecoration)
-IR_SIMPLE_DECORATION(AvailableInDXILDecoration)
 IR_SIMPLE_DECORATION(DownstreamModuleExportDecoration)
 
+struct IRAvailableInDownstreamIRDecoration : IRDecoration
+{
+    IR_LEAF_ISA(AvailableInDownstreamIRDecoration)
+    IRIntLit* getTarget() { return cast<IRIntLit>(getOperand(0)); }
```

Comment:
I recommend having a `CodeGenTarget getTarget()` accessor.
If there are places where we actually need the `IRIntLit` inst, we can have a separate `IRIntLit* getTargetInst()` accessor.

Reply:
Done